javascripts/discourse/components/featured-homepage-topics.js

```diff
@@ -13,12 +13,19 @@ export default class FeaturedHomepageTopics extends Component {
   @service siteSettings;
   @service currentUser;
   @tracked featuredTagTopics = null;
+  @tracked toggleTopics =
+    localStorage.getItem(this.toggleTopicsState) === "true" || false;
```

Comment:
I think its preferred for us to use keyValueStore, there is an example of a component using this here -> https://github.com/discourse/discourse-apple-add-to-homescreen/blob/94bcee284d97127f60ca07406eb9fc9bc35046ba/javascripts/discourse/components/add-to-homescreen.js#L13